tests/support/server.rs

```diff
@@ -23,6 +23,7 @@ pub struct Txn {
     pub read_timeout: Option<Duration>,
     pub response_timeout: Option<Duration>,
     pub write_timeout: Option<Duration>,
+    pub chunk_size: Option<usize>,
```

Comment:
Hm, you should get a stream of chunks as they are received, not at the end of the response, that's the point of the streaming interface. That is to say, even with a body that has a `Content-Length: 15`, if a read in hyper receives 5 bytes, it will yield a `Chunk` of those 5 bytes, and register to read more (still expecting 10 bytes).

However, it is also true that if the body is broken up by the chunked encoding, hyper will yield `Chunk`s each time it reaches the end of an encoding "chunk". So, if it received `5\r\nabcde\r\n1\r\nf\r\n`, it would yield a `Chunk` of 5, and then another poll would yield a `Chunk` of 1.